packages/venia-concept/src/components/ProductFullDetail/ProductFullDetail.js

```diff
@@ -131,54 +130,123 @@ const ProductFullDetail = props => {
-                    />
-                </Suspense>
-            </section>
-            <section className={classes.quantity}>
-                <h2 className={classes.quantityTitle}>Quantity</h2>
-                <Quantity initialValue={quantity} onValueChange={setQuantity} />
-            </section>
-            <section className={classes.cartActions}>
-                <Button
-                    priority="high"
-                    onClick={handleAddToCart}
-                    disabled={isAddingItem || isMissingOptions}
-                >
-                    Add to Cart
-                </Button>
-            </section>
-            <section className={classes.description}>
-                <h2 className={classes.descriptionTitle}>
-                    Product Description
-                </h2>
-                <RichText content={product.description} />
-            </section>
-            <section className={classes.details}>
-                <h2 className={classes.detailsTitle}>SKU</h2>
-                <strong>{product.sku}</strong>
-            </section>
+            {({ formState, formApi }) => (
+                <>
+                    <section className={classes.title}>
+                        <h1 className={classes.productName}>{product.name}</h1>
+                        <p className={classes.productPrice}>
+                            <Price
+                                currencyCode={productPrice.currency}
+                                value={productPrice.value}
+                            />
+                        </p>
+                    </section>
+                    <section className={classes.imageCarousel}>
+                        <Carousel
+                            images={mediaGalleryEntries.value}
+                            key={mediaGalleryEntries.key}
+                        />
+                    </section>
+                    <section className={classes.options}>
+                        <Suspense fallback={loadingIndicator}>
+                            <Options
+                                onSelectionChange={handleSelectionChange}
+                                product={product}
+                            />
+                        </Suspense>
+                    </section>
+                    <section className={classes.quantity}>
+                        <h2 className={classes.quantityTitle}>Quantity</h2>
+                        <DecrementButtonFormApi formApi={formApi} />
+                        <Text
+                            type="number"
+                            field="quantity"
+                            initialValue={quantity}
+                            onValueChange={setQuantity}
+                            validateOnChange
+                            validate={validateQuantity}
+                            className={classes.qtyfield}
+                        />
+                        <IncrementButtonFormApi formApi={formApi} />
```

Comment:
Would it be better from the usability and accessibility standpoint to have `+` `−` buttons grouped together after the qty input field? So the first thing on the tab path is the input? 

![stepper](https://user-images.githubusercontent.com/752610/59289749-74fa5500-8c3c-11e9-9d13-631214edcd3b.png)

Reply:
I think it's clearest as (minus, input, plus). The source order can be controlled with CSS so the tab order is not a concern, but normally I don't do that because it's equally weird to tab in retrograde motion. If the initial button (minus) has a useful value it should make sense when tabbing in any order.

The arrangement like `type="number"` with two small buttons has too small targets. This isn't as usable across devices, and there isn't a need to constrain the space.

Reply:
In my opinion (minus, input, plus) seems to be better enough. 
Because as in (minus, input, plus) minus and plus are separated by input so there can be less chances of confusion to user. 

Also I think as @brendanfalkowski suggested that two small buttons has too small targets, so we can prefer the (minus, input, plus) one.

Let me know your views, I can perform the changes as per the UI finalized.

Thanks.